src/Core/Resolvers/MultipleCreateOrderHelper.cs

```diff
@@ -61,6 +64,26 @@ public class MultipleCreateOrderHelper
                     subStatusCode: DataApiBuilderException.SubStatusCodes.EntityNotFound);
             }
 
+            DatabaseTable sourceDbTable = (DatabaseTable)sourceDbObject;
```

Comment:
How can we be sure the sourceDbObject is a DatabaseTable and not a view/stored proc? We should check for it before casting.

Reply:
Added